### PR TITLE
Add Methods for class Client

### PR DESCRIPTION
**Pull request by cohki0305**
Add methods 
  - http private 
  - http public

private methods' namespace is named after [Bitflyer API documents](https://lightning.bitflyer.jp/docs#http-private-api)

### CODE REVIEW

File: lib/bitflyer_api/methods/private/api.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module BitflyerApi
+  module Methods
+    module Private
+      module API
```
Comment (mc-chinju) [inline lib/bitflyer_api/methods/private/api.rb]: The nest is so deep.

I recommend to fix `module Bitflyer::Methods::Private::API`.

or

```ruby
module Bitflyer::Methods::Private
  module API
  end
end
```

Comment (cohki0305) [reply lib/bitflyer_api/methods/private/api.rb]: Fixed it!

**Comment by mc-chinju** [comment]
Thank you for your coding! 👍

**Comment by cohki0305** [comment]
Thank you for reviewing! I merge this pr.

**Pull request merged**